src/compose/types/page-block/progress.ts

```diff
@@ -67,6 +70,7 @@ export class PageBlockProgress extends PageBlock {
   }
 
   applyOptions (o?: Partial<Options>): void {
+    Apply(this.options, o, Number, 'refreshRate')
```

Comment:
After the !o return line!
Also double check for same problem in other files you changed. 

Apply should be one after the other and right after the !o return line